File: structure/stuff.py


def stuff(arr):
    myArr = list(set(arr))
    myTup = tuple(myArr)
    small = min(myTup)
    large = max(myTup)

    print('Uniques: {}'.format(myArr))
    print('')
    print('min: {}'.format(small))
    print('max: {}'.format(large))


sampleList = [87, 45, 41, 65, 94, 41, 99, 94]
sampleList2 = [87, 52, 44, 53, 54, 87, 52, 53]

stuff(sampleList)
stuff(sampleList2)

File: structure/stuff.js


const stuff = arr => {
    let a = [...arr];
    let uniques = [];

    while (a.length > 0) {
        let temp = a.shift();
        if (!a.includes(temp) && !uniques.includes(temp)) {
            uniques.push(temp);
        }
    }

    console.log('Uniques: ', uniques.join(' '));
    console.log('\n');
    console.log('min: ', Math.min(...arr));
    console.log('max: ', Math.max(...arr));
}


let sampleList = [87, 52, 44, 53, 54, 87, 52, 53];

stuff(sampleList);